root/layout/index.js

```diff
@@ -112,7 +112,7 @@ const Layout = (props) => {
 
         {!!server.alert &&
           <div className="banner warning-header">
-            <p>{server.alert}</p>
+            <p dangerouslySetInnerHTML={{__html: server.alert}}></p>
```

Comment:
The same issue applied to the alert message (which banner admins can set, and allows HTML).